### PR TITLE
Update README: Guidance for Customizing Upload Limits

### PR DESCRIPTION
**Pull request by ZarTek-Creole**
This commit adds a new section to the README.md file to provide users with guidance on how to customize upload limits in Lychee Docker container. This update is a follow-up to pull request https://github.com/linuxserver/docker-lychee/pull/64, where it was suggested that instead of changing default settings, it would be more beneficial to provide clear instructions to users on how to make this change themselves. 

The new section includes step-by-step instructions on modifying the `user.ini` file, as well as the necessary commands to restart the Docker container for the changes to take effect. It also adds a note of caution, reminding users that changing these settings may impact their server's performance.

### CODE REVIEW

**Review by nemchik — DISMISSED** [review]
Please note:
https://github.com/linuxserver/docker-lychee/blob/1bb561ce5bc9cb51c19cd3118b12089359ece9a1/README.md?plain=1#L1-L2

You need to edit the `app_setup_block` section here https://github.com/linuxserver/docker-lychee/blob/1bb561ce5bc9cb51c19cd3118b12089359ece9a1/readme-vars.yml#L84

**Comment by ZarTek-Creole** [comment]
I apologize for any confusion caused by the previous commits. This commit corrects the application setup instructions in the `readme-vars.yml` file. It provides guidance on customizing Lychee's upload limit to cater to individual server needs. I hope this addresses the concerns raised in the previous discussion, and I appreciate your patience and understanding. Thank you.

File: readme-vars.yml [hunk 1]
```diff
@@ -87,6 +87,35 @@ app_setup_block: |
   Setup mysql/mariadb and account via the webui, accessible at http://SERVERIP:PORT
   More info at [lychee]({{ project_url }}).
 
+  ## 🛠️ Customization
```
Comment (Roxedus) [inline readme-vars.yml]:
```suggestion
  ## Customization
```

File: readme-vars.yml [hunk 2]
```diff
@@ -87,6 +87,35 @@ app_setup_block: |
...
+
+  In certain scenarios, you might need to change the default settings of Lychee. For instance, if you encounter limitations when uploading large files, you can increase this limit. 📈
```
Comment (Roxedus) [inline readme-vars.yml]:
```suggestion
  In certain scenarios, you might need to change the default settings of Lychee. For instance, if you encounter limitations when uploading large files, you can increase this limit.
```

File: readme-vars.yml [hunk 3]
```diff
@@ -87,6 +87,35 @@ app_setup_block: |
...
+
+  ### 📤 Increasing Upload Limit
```
Comment (Roxedus) [inline readme-vars.yml]:
```suggestion
  ### Increasing Upload Limit
```

File: readme-vars.yml [hunk 4]
```diff
@@ -87,6 +87,35 @@ app_setup_block: |
...
+  The upload limit is defined in the `user.ini` file located in the config directory (`/config`). You can increase this limit by modifying the following values:
+
+  ```ini
+  post_max_size = 500M
+  upload_max_filesize = 500M
+  ```
+
+  After making these changes, you'll need to restart the Docker container for the changes to take effect. Here's how to do it:
+
+  #### 🔁 Via Docker Compose
```
Comment (Roxedus) [inline readme-vars.yml]:
```suggestion
  #### Via Docker Compose
```

File: readme-vars.yml [hunk 5]
```diff
@@ -87,6 +87,35 @@ app_setup_block: |
...
+
+  ```bash
+  docker-compose restart lychee
+  ```
+
+  #### 🔁 Via Docker Run
```
Comment (Roxedus) [inline readme-vars.yml]:
```suggestion
  #### Via Docker Run
```

File: readme-vars.yml [hunk 6]
```diff
@@ -87,6 +87,35 @@ app_setup_block: |
...
+
+  ```bash
+  docker restart lychee
+  ```
+
+  ⚠️ Please note that these changes might have implications on your server's performance, depending on its available resources. Thus, it's recommended to modify these settings with caution.
```
Comment (Roxedus) [inline readme-vars.yml]:
```suggestion
  **Please note that these changes might have implications on your server's performance, depending on its available resources. Thus, it's recommended to modify these settings with caution.**
```

**Review by Roxedus — CHANGES_REQUESTED** [review]
I would like to keep the usage of emojis unified across all our readmes.

File: readme-vars.yml [hunk 7]
```diff
@@ -87,6 +87,35 @@ app_setup_block: |
   Setup mysql/mariadb and account via the webui, accessible at http://SERVERIP:PORT
   More info at [lychee]({{ project_url }}).
 
+  ## Customization
```
Comment (Roxedus) [inline readme-vars.yml]:
```suggestion
  ### Customization
```

File: readme-vars.yml [hunk 8]
```diff
@@ -87,6 +87,35 @@ app_setup_block: |
...
+
+  In certain scenarios, you might need to change the default settings of Lychee. For instance, if you encounter limitations when uploading large files, you can increase this limit.
+
+  ### Increasing Upload Limit
```
Comment (Roxedus) [inline readme-vars.yml]:
```suggestion
  #### Increasing Upload Limit
```

File: readme-vars.yml [hunk 9]
```diff
@@ -87,6 +87,35 @@ app_setup_block: |
...
+  ```bash
+  docker-compose restart lychee
+  ```
+
+  #### Via Docker Run
+
+  ```bash
+  docker restart lychee
+  ```
+
```
Comment (Roxedus) [inline readme-vars.yml]: This should be self-explanatory
```suggestion
```

**Review by Roxedus — CHANGES_REQUESTED** [review]
Sorry, should have caught the levels of markdown sooner, this all belongs under `Application Setup`, which is at level 2

**Review by Roxedus — APPROVED** [review]

**Pull request merged**